api/src/test/java/uk/gov/dhsc/htbhf/claimant/communications/EmailMessagePayloadFactoryTest.java

```diff
@@ -73,19 +73,6 @@ void shouldBuildPaymentNotificationEmailPayloadWithAllPaymentTypes() {
         assertEmailPayloadCorrectForClaimantWithAllVouchers(payload.getEmailPersonalisation(), endDate.plusDays(1));
     }
 
-    @Test
-    void shouldBuildPaymentNotificationEmailPayloadWithOnlyPregnancyPayment() {
```

Comment:
This test didn't do anything that wasn't already being tested in the 3 tests above

Reply:
(It differed from shouldBuildEmailPayloadWithOnlyPregnancyPayment only by the EmailType)